docs/troubleshooting.md

```diff
@@ -0,0 +1,23 @@
+
+### I cannot join a worker node
+
+In [standalone](usage-overview.md#standalone) mode, check whether the join
+token has expired. If this is the case, attempt to create a new one using
+[JoinTokenRequest](https://docs.k0smotron.io/stable/join-nodes/#join-tokens) In
+[Cluster API](cluster-api.md), check the logs of your infrastructure provider controller.
+
+Also, check to see whether different versions were used in the initial cluster
```

Comment:
Maybe use "compatible" here? Kubernetes is ok with a limited number of different versions.

Reply:
Done.